pyseir/load_data.py

```diff
@@ -522,16 +523,29 @@ def get_current_hospitalized(state, t0, category):
     current estimate: float
         The most recent estimate for the current occupied in the requested category.
     """
-    MIN_DATAPOINTS_TO_CONVERT = 3  # We wait until the third datapoint to have 2 deltas to forecast
-
     abbr = us.states.lookup(state).abbr
     df = combined_datasets.build_us_timeseries_with_all_fields().get_data(
         AggregationLevel.STATE, country="USA", state=abbr
     )
 
-    categories = ["icu", "hospitalized"]
-    if category not in categories:
-        raise ValueError(f"Hospitalization category {category} is not in {categories}")
+    return _get_current_hospitalized(df, t0, category)
+
+
+def get_current_hospitalized_for_county(fips: str, t0: datetime, category: str):
+    df = (
+        get_hospitalization_data()
+        .get_subset(AggregationLevel.COUNTY, country="USA", fips=fips)
+        .get_data(AggregationLevel.COUNTY, country="USA", fips=fips)
+    )
+    return _get_current_hospitalized(df, t0, category)
+
+
+def _get_current_hospitalized(df: pd.DataFrame, t0: datetime, category: str):
```

Comment:
I think this function expects df to contain data from exactly one region (state or county). please clarify in a docstring.